kustomize/base/addons/storage/kustomization.yaml

```diff
@@ -1,2 +1,6 @@
 resources:
 - storage.yaml
+
+images:
+  - name: minio/minio
+    newTag: RELEASE.2021-04-06T23-11-00Z
```

Comment:
Can you move these lines to [this file](https://github.com/flyteorg/flyte/blob/master/kustomize/overlays/sandbox/kustomization.yaml#L24-L48).

Reply:
Done.